perllib/FixMyStreet/App/Controller/Report.pm

```diff
@@ -661,6 +661,33 @@ sub check_has_permission_to : Private {
     return \%permissions;
 };
 
+
+sub stash_category_groups : Private {
+    my ( $self, $c, $contacts ) = @_;
+
+    my %category_groups = ();
+    for my $category (@$contacts) {
+        my $group = $category->{group} // $category->get_extra_metadata('group') // [''];
+        # this could be an array ref or a string
+        my @groups = ref $group eq 'ARRAY' ? @$group : ($group);
+        if (scalar @groups < 2) {
```

Comment:
This now breaks the display when reporting a problem - because there you do want the existing behaviour of adding it to all the groups, you don't want a "Multiple Groups" option then. I think it has to be a flag passed to this for the two different behaviour.

Reply:
Oops, of course. We'll get there in the end :)